### PR TITLE
Removing rm -rf's

### PR DESCRIPTION
**Pull request by RaynardGerraldo**

### CODE REVIEW

**Comment by RaynardGerraldo** [comment]
> Looks good so far. Can you please also include the bugfix for Android from #391 ? The guy can't make a workable PR

yea im confused he just commited a "speedup download" , im just gonna try and implement what he did for the makefile

**Comment by RaynardGerraldo** [comment]
fixed @ura43

File: Makefile [hunk 1]
```diff
@@ -23,27 +23,28 @@ install:
 	chmod 0755 $(DESTDIR)$(PREFIX)/bin/ani-cli
 	echo "Installation successful (Mac OS)"
 uninstall:
-	rm -rf $(DESTDIR)$(PREFIX)/bin/ani-cli
+	rm $(DESTDIR)$(PREFIX)/bin/ani-cli
 	echo "Removal successful (Mac OS)"
 
 else ifeq ($(Platform), Android)
-TERMUX_BIN := /data/data/com.termux/files/usr/bin
```
Comment (ura43) [inline Makefile:30]: This one makes me worry. We should probably assign PREFIX to that, what do you think?

**Review by ura43 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (John-Russel) [reply Makefile:30]: PREFIX := /data/data/com.termux/files/usr/bin

Comment (ura43) [reply Makefile:30]: Well current master doesn't use the variable at all, but that could just mean that that got broken along the way too

Comment (John-Russel) [reply Makefile:30]: Termux already set PREFIX to /data/data/com.termux/files/usr
so adding PREFIX is useless i think

Comment (ura43) [reply Makefile:30]: > Termux already set PREFIX to /data/data/com.termux/files/usr so adding PREFIX is useless i think

wdym Termux set it? As in a default Termux environment has that variable by default?

Comment (RaynardGerraldo) [reply Makefile:30]: i tested it and it works with or without it

Comment (ura43) [reply Makefile:30]: then we can leave it out

Comment (CoolnsX) [reply Makefile:30]: > wdym Termux set it? As in a default Termux environment has that variable by default?

yup termux has that prefix by default.. and removing above line won't do any harm... I think this PR is good to go.

image

**Pull request merged**